### PR TITLE
Frontend tweaks

### PR DESCRIPTION
**Pull request by kponichtera**

### CODE REVIEW

File: frontend/src/gui/renderers.rs [hunk 1]
```diff
@@ -199,46 +206,87 @@ fn render_benchmark_instructions(ctx: &Context<AppRoot>, button_disabled: bool)
             <p style="padding-left: 2rem; padding-right: 2rem">
                 {"Please do "}
                 <strong>{"not"}</strong>
-                {" do anything else on your computer while running our benchmarks.
-                To ensure that you leave the tab open, you will have to press the button
-                at the bottom at least every 30 seconds. Press the START-button to start."}
+                {" do anything else on your computer while running our benchmarks,
+                so that the benchmark results are the most accurate.
+                Press the button below to start. Once the experiment finishes successfully,
+                click the button that appears to proceed to second experiment."}
```
Comment (Cornul11) [inline frontend/src/gui/renderers.rs:212]: Maybe it's better to specifically mention what the button is called?

File: frontend/index.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+.benchmark-progress-bar {
+    font-size: 1rem;
+}
```
Comment (Cornul11) [inline frontend/index.css:4]: For the sake of code maintainability, we can also try to add extract style attributes of general or specific elements to this file

**Review by Cornul11 — APPROVED** [review]

File: frontend/index.css (hunk 2 above)
Comment (Cornul11) [reply frontend/index.css:4]: To unclutter the rust definitions

Comment (kponichtera) [reply frontend/index.css:4]: I'm not doing that :joy:

Comment (Cornul11) [reply frontend/index.css:4]: Of course not, if we have time and nothing else to do, I could maybe do that later on

**Pull request merged**